### PR TITLE
[PRM-1017]: added API1812 feature switch

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -58,6 +58,11 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
     else etmpBase + "/"
   }
 
+  def getPenaltyDetailsUrl: String = {
+    if(!isEnabled(CallAPI1812ETMP)) stubBase + "/penalty/details/"
+    else etmpBase + "/penalties/details/"
```
Comment (LewisMitchell) [inline app/config/AppConfig.scala:63]: The URL should be /penalty/details/VATC/VRN/<vrn>

File: app/featureSwitches/FeatureSwitch.scala [hunk 2]
```diff
@@ -36,3 +36,7 @@ case object CallPEGA extends FeatureSwitch {
 case object CallDES extends FeatureSwitch {
   override val name: String = s"${FeatureSwitch.prefix}.call-des"
 }
+
+case object CallAPI1812ETMP extends FeatureSwitch{
+  override val name: String = s"${FeatureSwitch.prefix}.call-api-1812-etmp"
```
Comment (LewisMitchell) [inline app/featureSwitches/FeatureSwitch.scala:41]: An entry is needed in application.conf

File: app/utils/RegimeHelper.scala [hunk 3]
```diff
@@ -49,4 +49,6 @@ object RegimeHelper {
   }
 
   def constructMTDVATEnrolmentKey(vrn: String): String = s"HMRC-MTD-VAT~VRN~$vrn"
+
+  def constructMTDVATCUrl(vrn: String): String = s"VATC/VRN/$vrn"
```
Comment (LewisMitchell) [inline app/utils/RegimeHelper.scala:53]: I'm not sure what we are doing with this?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
I've raised some points here, however some may be invalidated when its rebased 😄

**Comment by BackRowOla** [comment]
QA and staging PR's
https://github.com/hmrc/app-config-qa/pull/14146/files
https://github.com/hmrc/app-config-staging/pull/9888

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**